wix/Lang/OpenJDK.dragonwell.de-de.wxl.template

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This whole file makes no sense to me. We have sentences in here like "AdoptOpenJDK Runtime Environment with Alibaba Dragonwell" and "AdoptOpenJDK JRE with Alibaba Dragonwell" and "JRE with Dragonwell". What's the naming concept again?

Reply:
The English version has the same problem.